src/js/lib/client/src/index.js

```diff
@@ -315,34 +315,42 @@ function createWindow() {
         webPreferences       : webPreferences,
         width                : windowCfg.width,
         height               : windowCfg.height,
-        frame                : false,
+        frame                : true,
         devTools             : false,
         sandbox              : true,
         backgroundThrottling : false,
-        vibrancy             : 'fullscreen-ui',
+        transparent          : false,
         backgroundColor      : "#00000000",
-        titleBarStyle        : 'hiddenInset'
+        titleBarStyle        : 'default'
     }
 
     if (args.dev) {
         windowPreferences.devTools = true
     }
 
+    // TODO [MM]
+    // Currently enabled to improve usability and dragging of window, fixes
+    // https://github.com/enso-org/ide/issues/743 as well as
+    // https://github.com/enso-org/ide/issues/484.
+    // If we don't want the default window frame,
+    // we can try to create our own solution in the future.
```

Comment:
No TODO comment needed here, create an issue instead for completing custom window frame and enabling it as default.